src/metaprob/autotrace.cljc

```diff
@@ -9,8 +10,8 @@
   [expressions stack]
   (map-indexed #(autotrace-expression %2 (cons %1 stack)) expressions))
 
-(defmacro autotrace #?(:clj [gen-expr] :cljs [env gen-expr])
-  (let [expr (expander/mp-expand #?(:cljs env) gen-expr)
+(defmacro autotrace [gen-expr]
+  (let [expr (expander/mp-expand &env gen-expr)
```

Comment:
Should be `#?(:cljs &env)`, as `mp-expand` doesn't expect an environment in Clojure :)